StringOps.java

```diff
@@ -1,42 +1,139 @@
 public class StringOps {
-    ////////////////////////////////////////////////////////////
-    //////                                               ///////
-    //////              Reminder:                        ///////
-    //////        allowed methods                        ///////
-    //////                                               ///////
-    //////        1.charAt(int index)                    ///////
-    //////        2.length()                             ///////
-    //////        3.substring(int start)                 ///////
-    //////        4.substring(int start,int ends)        ///////
-    //////        5.indexOf(String str)                  ///////
-    //////                                               ///////
-    //////        The rest are not allowed !             ///////
-    //////        if you want to use a different         ///////
-    //////        method, and you can implement          ///////
-    //////        it using material from the course      ///////
-    //////        you need to implement a version of     ///////
-    //////        the function by yourself.              ///////
-    //////                                               ///////
-    //////        see example for substring              ///////
-    //////        in Recitation 3 question 5             ///////
-    //////                                               ///////
-    ////////////////////////////////////////////////////////////
-    public static void main(String[] args) {
+    public static void main(String[] args)
+     {   
+        //String str = "MMMM"; 
+        //char ch = 'M'; 
+        //String strtest = "  Intro to coMPuter  sCIEnce"; 
+        String strtest = "One two tHRee world";
+        System.out.println(capVowelsLowRest(strtest)); 
+        //System.out.println(camelCase(strtest));
+        //int [] arrTEST = allIndexOf(str,ch);
+        //int length = arrTEST.length; 
+        //for (int i=0; i < length; i++)
+        //{
+           // System.out.print(arrTEST[i] );
+       // }
+
         
-    }
 
-    public static String capVowelsLowRest (String string) {
-        // Write your code here:
-        return "";
     }
 
-    public static String camelCase (String string) {
-        // Write your code here:
-        return "";
+    public static String capVowelsLowRest (String str)
+     {
+        String strNew = "";
+        int strlength = str.length();
+        int counter  =0; 
+        char ch = str.charAt(0);  
+        while (counter < strlength )
+        {
+            if (str.charAt(counter) == 'e' || str.charAt(counter) == 'a' 
+            || str.charAt(counter) == 'i' || str.charAt(counter) == 'o' ||  str.charAt(counter) == 'u')
+            {
+                 strNew =  strNew +  (char)(str.charAt(counter)- 32); 
+
+            }
+            else 
```

Comment:
when there is such a long if statement we tend to move into a function called for example isVowel( char )
and the code looks more readable.